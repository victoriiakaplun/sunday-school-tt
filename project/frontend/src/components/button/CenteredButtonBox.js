import React from 'react';

function CenteredButtonBox({ children }) {
  return <div className="field is-grouped is-grouped-centered">{children}</div>;
}

export default CenteredButtonBox;
