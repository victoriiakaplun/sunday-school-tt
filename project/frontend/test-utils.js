import React from 'react'; // eslint-disable-line no-unused-vars

import { initFontAwesomeLibrary } from './src/utils';

initFontAwesomeLibrary();

// import '@testing-library/jest-dom/extend-expect';

// re-export everything
export * from '@testing-library/react';
