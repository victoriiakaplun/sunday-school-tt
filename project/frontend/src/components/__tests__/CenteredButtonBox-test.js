import { render } from 'test-utils';

import CenteredButtonBox from '../button/CenteredButtonBox';

describe('CenteredButtonBox', () => {
  it('renders correctly', () => {
    const { container } = render(<CenteredButtonBox>Button</CenteredButtonBox>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
