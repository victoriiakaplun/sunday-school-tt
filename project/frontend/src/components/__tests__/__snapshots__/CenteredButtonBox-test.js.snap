// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenteredButtonBox renders correctly 1`] = `
<div
  class="field is-grouped is-grouped-centered"
>
  Button
</div>
`;

exports[`CenteredButtonBox renders correctly: 
      <div
       class="field is-grouped is-grouped-centered"
       >
      Column entry
      </div>
     1`] = `
<div
  class="field is-grouped is-grouped-centered"
>
  Button
</div>
`;
